wdl2cwl/main.py

```diff
@@ -43,57 +124,65 @@ def main(argv: List[str]) -> str:
     ast = WdlV1_1ParserVisitor()  # type: ignore
     ast.walk_tree(tree)  # type: ignore
 
+    input_types: List[str] = []
+    input_names: List[str] = []
+    input_values: List[str] = []
+
+    for i in ast.task_inputs:
+        input_types.append(i[0])
+        input_names.append(i[1])
+        input_values.append("None")
+
+    for i in ast.task_inputs_bound:
+        input_types.append(i[0])
+        input_names.append(i[1])
+        input_values.append(i[2])
+
     # returns the entire command including "command{........}"
     raw_command: str = cast(str, ast.task_command)
     raw_command = raw_command[
         raw_command.find("{") + 1 : -1
     ]  # removing the command{} part
-    command: List[str] = raw_command.strip().split("\\")  # split by '\'
-
-    raw_base_command = ""
-    base_command = []
-    command_arguments = []
 
-    for a in command:
-        # if it contains = , it's taken as an argument
-        if "=" in a or "~" in a:
-            command_arguments.append(a.strip())
-        # else add to base command
+    split_command: List[str] = raw_command.strip().split("\n")
+    command = ""
+    for i in range(0, len(split_command)):
+        split_command[i] = split_command[i].strip()
+        if "\\" in split_command[i]:
+            command += split_command[i]
+        elif "#" in split_command[i] or "\n" is split_command[i]:  # skip comments
+            continue
         else:
-            raw_base_command += a
+            command += split_command[i] + " "
 
-    # split the base command by spaces
-    base_command = raw_base_command.split()
+    command = get_command(
+        command, ast.task_inputs, ast.task_inputs_bound, input_types, input_names
+    )
+    print(command)
 
-    # get command arguments
-    index = 0
-    for i in command_arguments:
-        if "~" in i:
-            parameter_reference = i[
-                i.find("~{") + 2 : i.find("}")
-            ]  # finding the name of the parameter
-            sub_str = i.strip().split("~")
-            if "INPUT" not in sub_str[0]:
-                command_arguments[index] = (
-                    sub_str[0] + "$(inputs." + parameter_reference + ")"
-                )
-            else:
-                command_arguments[index] = (
-                    sub_str[0] + "$(inputs." + parameter_reference + ".path" ")"
-                )
-        index += 1
+    base_command = ["sh", "example.sh"]
 
     inputs = []
     for i in ast.task_inputs:
         input_type = wdl_type[i[0]]
         input_name = i[1]
         inputs.append(cwl.CommandInputParameter(id=input_name, type=input_type))
 
-    docker_requirement = [
+    requirements = []
```

Comment:
```suggestion
    requirements: List[cwl.ProcessRequirement] = []
```
Because we did not annotate `requirements` with a type hint, the type checker guess that it should be a `List` of `cwl.DockerRequirement`s based upon the first usage of `requirements` (line 173).

So if we say that this is a `List` of `cwl.ProcessRequirement` (the parent type of `cwl.DockerRequirement` and `cwl.InitialWorkDirRequirement`) then that should clear up the error

Reply:
The PR is still failing. What more can I do? I've fixed the errors mentioned in the main.py file. 
```ERROR:   py36-lint: commands failed```
```ERROR:   py37-lint: commands failed ```
I get these errors. Do they have any effect?